specs/latest/2.0/index.html

```diff
@@ -3527,13 +3527,16 @@ <h3><a name="CLIENT_WAIT_SYNC">clientWaitSync</a></h3>
 
     <h3><a name="VERTEX_ATTRIBUTE_ALIASING">Vertex Attribute Aliasing</a></h3>
     <p>
-        In the WebGL 2.0 API, if more than one active attribute name is bound to the same location, it is considered
-        aliased, regardless whether there exists a path through the shader that consumes more than of these
-        attributes. Such definition of aliasing is broader than what is defined in
-        <span class="gl-spec">(<a href="http://www.khronos.org/registry/gles/specs/3.0/es_spec_3.0.4.pdf#nameddest=section-2.11.5">OpenGL ES 3.0.4 &sect;2.11.5</a>)</span>.
+        The WebGL 2.0 API relaxes the restrictions in GLSL ES 3.00.6 section 12.46. In the WebGL 2.0
+        API, two attributes that are bound to the same location are only considered to be aliased in
+        case they are both statically used. Otherwise the resolution of GLSL ES 3.00.6 section 12.46
```

Comment:
We don't want to be more forgiving than our sources spec here. This won't be the only place where webglsl3 diverges from webglsl1.

Reply:
It's one less thing that may break WebGL1 code when upgrading to WebGL2.

We already have the code for implementing WebGL1 on top of ES3 drivers. So extending that to WebGL2 should be trivia.

Reply:
No, I don't want to do this. Pulling forward restrictions is something we should be avoiding.

Reply:
It's a little tiny effort on browser vendors' side, but could make upgrading WebGL 1 contents to WebGL 2 experience smoother, which is an important goal.